### DISCUSSION TITLE
Meetings listed twice, coming from (2) Intergroups

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#1358] [Q&A]
**Dear C4R Support,**

I was asked by the Reading-Berks Intergroup Web Servant why some meetings that are listed on the Area 59 website were being listed twice on the app. I began by asking him if he was the person maintaining the source document.

He sent me this email in reply:

> I am not aware of any master list. If there ever was one, I’ve not been privy to it.
> I’ve been using WordPress with the TSML plug-in since taking over from Greg when he got sick.  Once I generated the API key with some help from the TIAA forum, it worked quite well.
> 
> Whenever I’ve added, edited or deleted info or an entire meeting, it has been updated on the app & A59 website without issue 98% of the time.
> 
> The only previous problems have occurred when Elementor & the TSML plug-in have had simultaneous updates, which has occurred 2x in the last 3 years. 
> 
> Does anyone at Area59 know why some meetings show up as A59 being the source? I’ve only communicated about the ones I’ve seen that reside in Berks County, and this “Came to Believe” meeting & what just happened with “A Fresh Start” isn’t the first I’ve seen it occur. I’ve also seen A59 as the source for other meetings in other counties as well. Some even show up twice in the app, once for the correct source location, & once from “Area59 Pottsville PA. 
> 
> I’m attaching an example that is currently on the app. It’s the same meeting, however one says it’s from SEPIA, the other from Area 59. 

The example he gave: [https://area59aa.org/find-a-meeting/?region=richlandtown](https://area59aa.org/find-a-meeting/?region=richlandtown)

![IMG_992771BCD7CF-1](https://github.com/code4recovery/12-step-meeting-list/assets/95769327/42df4e86-f1bf-42f7-879b-22bc9f0e6e08)

![IMG_0E239F529DF4-1](https://github.com/code4recovery/12-step-meeting-list/assets/95769327/b60de0c9-298a-4d9f-a275-050ece3a58b8)

Area 59 uses feeds from two geographically overlapping Intergroups on our site: Reading-Berks and SEPIA.

[https://readingberksintergroup.org](https://readingberksintergroup.org)

[https://area59aa.org](https://area59aa.org)

[https://area59aa.org](https://area59aa.org)

![two-intergroup-feeds](https://github.com/code4recovery/12-step-meeting-list/assets/95769327/2a5a34df-8f52-41d0-868e-015b98274154)

Should Area 59 disconnect the feed from Reading-Berks Intergroup?

Any help would be appreciated.

Thank you,

**Scott Fritzinger**  |  Area 59 Web Servant  |  webservant@area59aa.org

**Comment by joshreisner** [comment]
hi scott, hard to know without more details about the meetings in question. is it an in-person meeting? these are supposed to be de-duplicated by address. what's the address listed in the app for these meetings?

**Reply by joshreisner** [reply]
@fritzisa email attachments don't show up on github. please come to the discussion topic and upload it, or, better for searching, just add the information as text. thanks

**Comment by fritzisa** [comment]
Hi, Josh,

Please see the attached 2-page PDF that includes Steve’s email to me and some supporting images and links.

It looks like Area 59 and Southeastern PA Intergroup (SEPIA) are both using outdated feeds from Reading-Berks Intergroup.

Steve took over the Reading-Berks Intergroup website and meeting list when Greg G., their former Web Servant, became ill.

I believe that was about the time the feeds were last updated (Dec. 2021).

I’ve included Steve (cc’d) in this email.

Any guidance you can provide would be greatly appreciated.

In fellowship,

Scott Fritzinger  |  Area 59 – Web Servant  |  (610) 762-7792


> On Feb 17, 2024, at 8:48 PM, Josh Reisner ***@***.***> wrote:
> 
> 
> hi scott, hard to know without more details about the meetings in question. is it an in-person meeting? these are supposed to be de-duplicated by address. what's the address listed in the app for these meetings?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/code4recovery/12-step-meeting-list/discussions/1358#discussioncomment-8504290>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AW2VF3ZC5PYARDPPR3KUQV3YUFMYRAVCNFSM6AAAAABDNUKS4GVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMBUGI4TA>.
> You are receiving this because you authored the thread.
> 
￼

**Reply by fritzisa** [reply]
[Reading-Berks-Intergroup-Assistance-02-18-2024.pdf](https://github.com/code4recovery/12-step-meeting-list/files/14323893/Reading-Berks-Intergroup-Assistance-02-18-2024.pdf)

**Reply by fritzisa** [reply]
FEB 15, 2024
FROM: Reading-Berks Web Servant, Steve P. d33adcm@readingberksintergroup.org
TO: Area 59 Web Servant, Scott Fritzinger webservant@area59aa.org

I changed the status of this meeting on February 3 to show it cant be attended in person. That update has not showed up yet on the Area 59 Meeting schedule or the meeting guide app 2 weeks later. I also placed an asterisk in front of the listing for my home group on the same day. That HAS shown up on the area 59 meeting schedule & the meeting guide app.

While sniffing around I noticed that the listing for a Fresh Start no longer says it is from Reading Berks Intergroup. It says it’s from Area 59 Pottsville PA. The update I made for both show up correctly on the Reading Berks Intergroup website, which is the source for the Area 59 info from our county. I’m uncertain what to do about not being able to get the meeting app to show the correct info. For some reason it seems to be using the data from Area 59, not Reading-Berks Intergroup.
Any suggestions?

Steve P
RBI web servant 
-----
Public URL: https://area59aa.org/find-a-meeting/?meeting=a-fresh-start-meeting
Edit URL: https://area59aa.org/wp-admin/post.php?post=2573738&action=edit

![screenshot-from-Steve-P-FEB-15-2024](https://github.com/code4recovery/12-step-meeting-list/assets/95769327/d8da3c1d-e2ff-4d4a-849f-8006588ba14c)

Reading-Berks Intergroup website:
https://readingberksintergroup.org

Area 59 website:
https://area59aa.org

Southeastern Pennsylvania Intergroup website:
(SEPIA)
https://aasepia.org

Richlandtown Candlelight
Saturday, 7:00 PM to 8:00 PM
St. John’s Lutheran Church (D47)
4 South Main Street, Richlandtown, PA

<img width="803" alt="Screenshot 2024-02-18 at 2 19 45 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/95769327/eb4b4ca9-0116-4ea5-b5d5-5e3557390362">

**Reply by joshreisner** [reply]
thanks @fritzisa super helpful. so with Richlandtown Candlelight, the meeting is listed twice because there are (slightly) different addresses. SEPIA lists what looks like the correct address: [4 S Main St](https://www.aasepia.org/meetings/?meeting=meeting-4-s-main-st-richlandtown-pa-18955-usa-6-190000), whereas the Area is listing the address as `4, 4 S Main St` - Meeting Guide sees two different addresses. do you know if the area is importing this meeting? i think if it can be corrected at the source it should start being listed only once

Re A Fresh Start, looks like it's currently listed by the Area in the app - however the data appears to be consistent between the area and the intergroup, so it looks like the only issue is who it appears to be listed by in the app -- if RBI would prefer it be listed by them in the app we are working on an update that should accommodate this with no further action on your part. however if you want to "fix" it right now you could stop importing from them, and, because they are already connected, the meeting will start being listed by them directly

lmk if i missed anything or if you have other questions

**Reply by fritzisa** [reply]
**Thanks very much, Josh!**

So, I will disconnect the RBI feed from Area 59.

It looks like SEPIA was also connected to the feed RBI had previously been sharing with Area 59.

Should Steve ask SEPIA to do the same thing?

I believe when Steve took over in December 2021 he connected the RBI meeting information with the app.

Which made the feed shared with Area 59 and SEPIA no longer needed.

That would explain why our last refresh was December 2021 on both Area 59 and SEPIA.

We appreciate all your help!

In fellowship,

**Scott Fritzinger**  |  Area 59 – Web Servant  |  (610) 762-7792

**Reply by joshreisner** [reply]
hm, i don't think RBI lists the richmontown candlelight meeting, and also i don't think SEPIA imports feeds from anyone. 

i think it's ok that the meeting listing hasn't been updated in that long, it just means there hasn't been a change, hopefully

**Reply by fritzisa** [reply]
Oh, I thought Richlandtown Candlelight was one of RBI's meetings, and that's why he brought it up.

Out of curiosity, how does a big Intergroup like SEPIA keep track of all the meetings in all their Districts?

I really appreciate your patience and time in helping me understand this stuff.

**Reply by joshreisner** [reply]
i don't know much specifically about SEPIA, but intergroups of that size in general use a few techniques, usually they will have volunteers and paid staff who make updates and answer phone calls - the latter can be very helpful gathering reports of missing meetings. they often there will be an "Outreach Committee," or something similar, to track down and contact or sometimes visit the meetings in question. they'll also have a database of meeting contacts - these are usually just the names of the people who contacted the intergroup last to add or edit the meeting info.

however i wouldn't be too spooked about the late 2021 updated date, it's not unusual for a meeting's listing to stay the same for several years. if it had been early 2021 i'd be more concerned. 

if you're importing richlandtown from SEPIA perhaps your site could use an address override to prevent it from being listed twice?

```
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '4 S Main St, Richlandtown, PA 18955, USA' => array(
            'formatted_address' => '4 S Main St, Richlandtown, PA 18955, USA',
            'city' => 'Richlandtown',
            'latitude' => 40.4706128,
            'longitude' => -75.3207652,
            'approximate' => 'no',
        ),
    ));
}
```

**Reply by joshreisner** [reply]
@fritzisa i don' think you needed to contact SEPIA, it's your site that's listing the address incorrectly. this is happening because of a quirk of Google geocoding. if you can add the code above that should hopefully fix it. if not, you might need to also delete and reimport their feed after adding the code.

**Reply by fritzisa** [reply]
If it were our site that's listing the address incorrectly, wouldn't I be able to edit the address? I tried, and it said it wasn't ours to edit.
Can you give more more detailed instructions about where to add the code you provided?

**Reply by joshreisner** [reply]
yes sorry you can add that code to your theme's functions.php file. 

you can try editing the address without adding it, but i don't think it will work, because the plugin checks with google what it thinks the correct address is and they are sending back `4, 4 S Main St, Richlandtown, PA 18955, USA`, which seems wrong

**Comment by fritzisa** [comment]
Thanks very much, Josh!

So, I will disconnect the RBI feed from Area 59.

It looks like SEPIA was also connected to the feed RBI had previously been sharing with Area 59.

Should Steve ask SEPIA to do the same thing?

I believe when Steve took over in December 2021 he connected the RBI meeting information with the app.

Which made the feed shared with Area 59 and SEPIA no longer needed.

That would explain why our last refresh was December 2021 on both Area 59 and SEPIA.

We appreciate all your help!

In fellowship,

Scott Fritzinger  |  Area 59 – Web Servant  |  (610) 762-7792

**Reply by fritzisa** [reply]
To address the Richlandtown Candlelight meeting being listed twice, I contacted SEPIA using their Meeting Update page of their website.
Form on SEPIA site:
[https://www.aasepia.org/meeting-status/](https://www.aasepia.org/meeting-status/)
Here is a copy of the request:

Dear SEPIA,
I am the Web Servant for Area 59, and we use the SEPIA meeting feed on our website: https://area59aa.org/
I was notified by Steve P., Reading-Berks Intergroup Web Servant that there are (2) listings for this meeting.
His email address:  [d33adcm@readingberksintergroup.org](mailto:d33adcm@readingberksintergroup.org)
The only difference is an apparent typo in the street address where the building number is listed twice:
Incorrect listing:  4, 4 S Main St, Richlandtown, PA 18955, USA
Should be:            4 S Main St, Richlandtown, PA 18955, USA
Please acknowledge receipt of this request to me and Steve.
Thank you,
Scott Fritzinger, Area 59 Web Servant, [webservant@area59aa.org](mailto:webservant@area59aa.org)
